projects/indicators/src/modules/chevron/chevron.component.html

```diff
@@ -1,10 +1,9 @@
 <button
   class="sky-chevron"
   type="button"
-  [attr.aria-label]="
-    (direction === 'down' ? 'skyux_chevron_expand' : 'skyux_chevron_collapse')
-      | skyLibResources
-  "
```

Comment:
> Bad a11y is worse than no a11y

I'm considering the "expand/collapse" labels a bug, as they didn't tell the screen reader what is being expanded/collapsed. Going forward, we need to make sure we're providing proper labels via the `ariaLabel` input.  Since this is an internal component, I've filed bugs for each component that uses them that needs to be updated.